### DISCUSSION TITLE
【公告】绳网以于地球Online正式上线

### DISCUSSION THREAD

**Discussion by Dreamlsk** [share121/inter-knot#728] [其他]
![QQ20240810-154008](https://github.com/user-attachments/assets/3bf65c7b-cc0f-4da1-871a-20880b46af5e)
尊敬的地球Online玩家，绳网已正式上线。
                                           祝您游戏愉快。

**Comment by cheeseRatRat** [comment]
可以登出地球Online吗？能告诉我登出键在什么地方吗？？

**Reply by Dreamlsk** [reply]
亲爱的玩家：
你好。游戏：地球Online 因沉浸式体验而闻名，暂不支持登出。这一点在进入游戏前的用户协议中有提到。
玩家只能通过正常在游戏中进行游玩直至游戏人物死亡而退出。感谢支持，祝您游戏愉快。

**Comment by AusertDream** [comment]
登出键已经被扣掉了ヽ(`Д´)ﾉ

**Comment by yongko2006** [comment]
monkey！

**Comment by GanlinSama** [comment]
听说美服那边又要换公会领导人了

**Reply by Kaohaaa** [reply]
为什么我一到朝鲜就切换到单机模式了？

**Reply by Callkit-P0p** [reply]
朝鲜服务器网络设备更新太慢了，已经落后其他服好几个版本了 索性就改成单机减少设备更新费用

**Comment by Callkit-P0p** [comment]
**16**年地球_Online_老玩家路过

**Comment by SDCOM-0415** [comment]
我要注销我的地球ol账号了
